upd/MaprDBUpdater.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/urfave/cli.v2"
+	"log"
+	"os"
+	"sync"
+	"vmrenter/pkg/config"
+	"vmrenter/pkg/mapr"
+	"vmrenter/pkg/models"
+	"vmrenter/pkg/utils"
+)
+
+var nodesTable = "/user/mapr/nodes"
+
+func update(c *cli.Context) error {
```

Comment:
This function is pretty long. I would try to split it in smaller functions.
The funny thing is it's almost already split.
Every comments seems to split the code in smaller steps.
- Getting nodes from csv file
- Creating NodeDBJsons from nodes
- Updating NodeDBJsons with ClusterID, ExpiresAT
- Resetting /user/mapr/node table
- Updating the nodes table

Reply:
I would leave getting nodes, resetting nodes table unextracted as they don't contain a lot of logic but just several prints, agree?

Reply:
@alaingilbert please check the fix